src/base/binding.coffee

```diff
@@ -64,11 +64,23 @@ qdInternal.binding = {
             # pass the real dom node to the binding function as the $element value
             return bindingFunction(bindingContext, qdInternal.dom.unwrap(domNode))
         catch err
+            ancestorPath = @_getNodeAncestry(domNode)
             error = new QuickdrawError("'#{err.message}' in binding '#{domNode.getAttribute(qd.getConfig('bindingAttribute'))}'", err)
             error.setBindingContext(bindingContext)
+            error.setAncestorPath(ancestorPath)
             qdInternal.errors.throw(error)
             return null
 
+    # Gets the tagName and className for all nodes in the ancestry chain of this VirtualDomNode
+    # @param [VirtualDomNode] domNode the node to calculate the ancestry chain of
+    # @return [String] the ancestry chain of the provide domNode
+    _getNodeAncestry : (domNode) ->
+        ancestorPath = []
+        while domNode?
+            ancestorPath.push(domNode.getProperty('tagName').toLowerCase() + "." + domNode.getProperty('className').replace(" ", "."))
```

Comment:
If a node doesn't have a class set, does `domNode.getProperty('className')` return `null` or an empty string?

Reply:
Looks like it returns an empty string.  If I'm reading the code right, if it's not in the custom properties, it pulls it right off the node.  

    https://github.com/hulu/quickdraw/blob/76962bbbf13973713e8a2bc10dfc24fea014b2e4/src/base/dom.coffee#L203

And that in my little test returns an empty string:

    document.createElement("div")["className"] -> ""